packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -53,9 +53,6 @@ export class OdspDeltaStorageService {
             let postBody = `--${formBoundary}\r\n`;
             postBody += `Authorization: Bearer ${storageToken}\r\n`;
             postBody += `X-HTTP-Method-Override: GET\r\n`;
-            if (fetchReason !== undefined) {
-                postBody += `fetchReason: ${fetchReason}\r\n`;
-            }
 
```

Comment:
Do we need to leave this here for compatibility while the service switches over to looking in the header?

Reply:
There are not logging yet. The work item was pending on their end that is why they requested this change before starting their change.